src/pages/Task.tsx

```diff
@@ -103,6 +107,7 @@ export const Task: React.FC = () => {
         }
         buttons={
           <ActionButtons
+            isPatchOnCommitQueue={isPatchOnCommitQueue}
             canAbort={canAbort}
             canRestart={canRestart}
```

Comment:
Instead of adding another prop could we just update the existing props? 
`canRestart={!isPatchOnCommitQueue && canRestart}`

Reply:
yes! done! Checked with `yarn prod` that it still works.